lib/api/editObject.js

```diff
@@ -22,6 +23,15 @@ function objectIsValid (object) {
   return object && object !== ''
 }
 
+function tripleIsValid (subject, predicate, object) {
+  if (predicate === PREDICATES['role']) {
```

Comment:
This requires additional validation. I am not sure how to do this. @cjmyers -- ideas?

Reply:
The only validation required is that the provided role is a URI.  I wonder if you try to convert the string entered to a URI, if maybe it will give you an exception if the string is not a valid URI string.